content/docs/reference/validation_errors@es.md

```diff
@@ -389,10 +389,10 @@ Hacen referencia a etiquetas previstas:
 Este error se produce cuando una etiqueta contiene dos atributos que se excluyen mutuamente.
 Por ejemplo, en los siguientes casos, solo se permite uno de los dos atributos:
 
-* [amp-twitter](/docs/reference/extended/amp-twitter.html): `data-tweetid` o `src`
-* [amp-instagram](/docs/reference/extended/amp-instagram.html): `data-shortcode` o `src`
-* [amp-iframe](/docs/reference/extended/amp-iframe.html): `src` o `srcdoc`
-* [amp-youtube](/docs/reference/extended/amp-youtube.html): `src` o `data-videoid`
+* [amp-twitter](/es/docs/reference/extended/amp-twitter.html): `data-tweetid` o `src`
```

Comment:
Links to the component's reference docs will break because there are no redirects for `<lang>/docs/reference/extended/<amp-component>.html`.

Use this `/es/docs/reference/component/<amp-component>.html`
-OR-
leave as is, since the  amp component reference docs are all in English.